src/types.flow.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+export type A11yRole =
```

Comment:
Can we add a reference link? https://reactnative.dev/docs/accessibility#accessibilityrole-ios-android
Ideally we could import directly from `react-native`, but I'm not sure if it's exported and if it would be backward compatible with older RN versions. This applies to TS as well